web/themes/custom/urbanobserver/components/02-molecules/menus/navigation/navigation-menu.js

```diff
@@ -1,31 +1,37 @@
 Drupal.behaviors.mainMenu = {
   attach(context) {
-    const toggleExpand = context.getElementById('toggle-expand');
-    const menu = context.getElementById('navigation-nav');
-    const logo = context.getElementById('nav-logo');
-    const littlePreMenu = context.getElementById('lower-block');
-    if (menu) {
-      const expandMenu = menu.getElementsByClassName('expand-sub');
+    const toggleExpand = once('toggle-expand', '#toggle-expand', context);
+    const menu = once('navigation-nav', '#navigation-nav', context);
+    const logo = once('nav-logo', '#nav-logo', context);
+    const littlePreMenu = once('lower-block', '.little-pre-menu', context);
+    const searchIcon = once('search-icon', '#search-icon', context);
+    const searchForm = once('search-block-form', '#block-urbanobserver-search-form-wide', context);
+    const pricingButton = once('see-pricing', '.button-see-pricing', context);
+    const socialLinks = once('social-links', '#social-links', context);
```

Comment:
There is a lot of "once" here. Are all of them really needed?
Logic is probably working, but it is not right if all these elements are related.

If all of them are always under a single element, it is better to get that element with a single one, and then find others from it.
Probably something similar can be done with an "open" class on each element.